build/Dockerfile

```
@@ -307,10 +307,15 @@ apt-get install -y --no-install-recommends \
     libboost-filesystem1.81.0 \
     libboost-log1.81.0 \
     libcrypto++8 \
-    procps
+    procps \
+    nodejs \
+    npm
 rm -rf /var/lib/apt/lists/*
 EOF
 
+RUN npm install -g postgraphile
+RUN npm install -g --save @graphile-contrib/pg-simplify-inflector
```

```suggestion
rm -rf /var/lib/apt/lists/*
npm install -g postgraphile
npm install -g --save @graphile-contrib/pg-simplify-inflector
EOF
```

These lines could be inside the previous RUN